pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1240,7 +1240,7 @@ private void verifyIndexingConfig(String tableNameWithType, IndexingConfig index
     }
   }
 
```

Comment:
Since we made this a public method, can you add some javadocs explaining what this method does?